### PR TITLE
Unfuck brainfart

### PR DESCRIPTION
**Pull request by fribmendes**

### CODE REVIEW

**Comment by fribmendes** [comment]
@joaofcosta can you double check this works? Don't have the test data for it.

**Comment by joaofcosta** [comment]
@fribmendes I'll check it tomorrow morning with the test data I have. Seems like a really nice little brainfart #classy :+1:

File: app/models/activity.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 class Activity < ActiveRecord::Base
   has_many :registrations
-  has_many :users, through: :registrations
+  has_many :participants, through: :registrations,
+                          class_name: "User"
```
Comment (joaofcosta) [inline app/models/activity.rb]: @fribmendes Using `class_name: "User"` I get the following error: `Could not find the source association(s) "participant" or :participants in model Registration. Try 'has_many :participants, :through => :registrations, :source => <name>'. Is it one of activity or user?`. Changing it to `source: :user` fixes the problem :+1:

Comment (fribmendes) [inline app/models/activity.rb]: @joaofcosta done

**Pull request merged**